docs/pages/machine-id/access-guides/ssh.mdx

```diff
@@ -3,4 +3,155 @@ title: Machine ID with Server Access
 description: How to use Machine ID to access servers via SSH
 ---
 
-TODO
\ No newline at end of file
+(!docs/pages/includes/machine-id/v2-config-warning.mdx!)
+
+Teleport Server Access protects and controls SSH access to servers. Machine ID
+can be used to grant machines secure, short-lived access to these servers.
```

Comment:
```suggestion
Teleport protects and controls SSH access to servers. Machine ID
can be used to grant machines secure, short-lived access to these servers.
```

Since we're no longer describing "<Resource> Access" features as distinct products (outside docs titles where we're using title case).